docs/docker.md

```diff
@@ -35,8 +35,16 @@ chain:
 mining:
     autostart: true
     beneficiary: "encoded_beneficiary_pubkey_to_be_replaced"
+    cuckoo:
+        miner:
+            executable: lean30
```

Comment:
```suggestion
            executable: lean29-generic
```

Reply:
If avx2 is available use `lean29-avx2`

Reply:
Since I don't really know about those miners, I'll just apply the suggestion.